test/packageLogs_spec.js

```diff
@@ -313,28 +313,15 @@ describe('packageLogs', () => {
     describe('buildPath', () => {
         it('builds a path', () => {
             new JSDOM(``)
+            let actualPath
             const document = window.document;
             const ele = document.createElement('div');
-            const evt = document.createEvent('CustomEvent');
-            evt.initEvent('testEvent', true, true);
+            const evt = new window.Event('CustomEvent', {bubbles: true, cancelable: true})
             document.body.appendChild(ele);
+            ele.addEventListener('CustomEvent', e => actualPath = buildPath(e))
             ele.dispatchEvent(evt);
-            expect(buildPath(evt)).to.deep.equal(['div', 'body', 'html']);
-        });
-
-        it('defaults to path if available', () => {
-            new JSDOM(``)
-            const document = window.document;
-            const ele = document.createElement('div');
-            const evt = document.createEvent('CustomEvent');
-            document.body.appendChild(ele);
-            evt.initEvent('testEvent', true, true);
-            ele.dispatchEvent(evt);
-            evt.composedPath = function() {
-                let ele = evt.target;
-                return [ele, ele.parentElement, ele.parentElement.parentElement];
-            };
-            expect(buildPath(evt)).to.deep.equal(['div', 'body', 'html']);
+            const expectedPath = ['div', 'body', 'html',   "#document", "Window"]
```

Comment:
Noting here that the `expectedPath` now includes `"#document"` and `"Window"`. This is expected behavior with `composedPath`, see screenshot from Chrome dev tools below:
<img width="554" alt="Screen Shot 2023-01-02 at 11 14 13 AM" src="https://user-images.githubusercontent.com/42304551/210262096-5436c066-57a9-481c-906f-85069f0e8fae.png">

Reply:
@UncleGedd AWESOME. Thanks, Gedd. Was planning some maintenance next week... I'll beat on this a bit and report back!